Stytch/Source/SMS Auth/StytchOTP.swift

```diff
@@ -0,0 +1,99 @@
+import UIKit
+/*
+@objc private protocol StytchMagicLinkImpl {
+    @objc static var shared: StytchMagicLink { get }
+    @objc func configure(projectID: String, secret: String, scheme: String, host: String)
+    @objc func configure(projectID: String, secret: String, scheme: String, host: String, universalLink: String)
+    @objc var `debug`: Bool { get set }
+    @objc var environment: StytchEnvironment { get set }
+    @objc var loginMethod: StytchLoginMethod { get set }
+    @objc func handleMagicLinkUrl(_ url: URL?) -> Bool
+    @objc func login(email: String)
+}
+*/
+@objc(SAStytchOTPAuthenticator) public protocol StytchOTPAuthenticator {
+    //@Ethan I probably need to provide more than just the code to clients in order for them to auth.
+    //"method_id": "phone-number-test-d5a3b680-e8a3-40c0-b815-ab79986666d0",
+    //"code": "123456"
+    func authenticateOTP(_ code: String, methodId: String, success: @escaping (AuthenticatedOTPResponse) ->(), failure: @escaping (StytchError) ->())
```

Comment:
Switch out AuthenticatedOTPResponse for empty closure. Maybe pass in the error as an optional error as well.

Reply:
Also add a custom error type with an associated String type.